README.md

```diff
@@ -437,50 +437,68 @@ tabular_explainer = TabularExplainer(clf.steps[-1][1],
-|
+* Model Performance
+* Data Explorer	
+* Aggregate Feature Importance
+* Individual Feature Importance and What-If	
 
+>[!NOTE]
+> Click on "Open in a new tab" on the top left corner to get a better view of the dashboard in a new tab.
 
-![Visualization Dashboard Global](https://docs.microsoft.com/en-us/azure/machine-learning/service/media/machine-learning-interpretability-explainability/global-charts.png)
 
+You can further create custom cohorts (subgroups of your dataset) to explore the insights across different subgroups (e.g., females vs males). The created cohorts can contain more than one filter (e.g., age < 30 and sex = female) and will be visible from all of the four tabs. The following sections demonstrate the visualization dashboard capabilities on a [classification model trained on employee attrition dataset]((https://github.com/interpretml/interpret-community/blob/master/notebooks/simple-feature-transformations-explain-local.ipynb)). Besides the default cohort (including the whole dataset), there are two additional cohorts created: employees with Age <= 35 and employees with Age > 35.
 
 
-### Local visualizations
+![Visualization Dashboard Cohorts](./img/Cohorts.png)
 
-You can click on any individual data point at any time of the preceding plots to load the local feature importance plot for the given data point.
 
+### Model performance 
+This tab enables you to evaluate your model by observing its performance metrics and prediction probabilities/classes/values across different cohorts.
+
+![Visualization Dashboard Cohorts](./img/ModelPerformance.png)
+
+### Dataset explorer
+You can explore your dataset statistics by selecting different filters along the X, Y, and color axis of this tab to slice your data into different dimensions.
+
+![Visualization Dashboard Cohorts](./img/DatasetExplorer.png)
+
+The following plots provide a global view of the trained model along with its predictions and explanations.
+
+### Aggregate feature importance (global explanation)
+
+This view consists of two charts:
 |Plot|Description|
 |----|-----------|
-|Local Importance|Shows the top K (configurable K) important features for an individual prediction. Helps illustrate the local behavior of the underlying model on a specific data point.|
-|Perturbation Exploration (what if analysis)|Allows changes to feature values of the selected data point and observe resulting changes to prediction value.|
-|Individual Conditional Expectation (ICE)| Allows feature value changes from a minimum value to a maximum value. Helps illustrate how the data point's prediction changes when a feature changes.|
+|Feature Importance| Explore the top-k important features that impact your overall model predictions (a.k.a. global explanation). Use the slider to show descending feature importance values. Select up to three cohorts to see their feature importance values side by side.|
```

Comment:
Maybe just "Use the slider to show additional less important feature values." or something like that? Descending feature importance values is confusing to me.